docs/docs/monitoring/load-testing-guidelines.mdx

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Load Testing Guidelines
+title: Load Testing Guidelines
+description: |
+  Information about how best to scale up your bot to support more parallel user activity
+  and how you can use tracing to help debug issues.
+---
+
+:::note
+
+Our team is currently in the process of running more extensive performance related tests, more information will be added here as we progress.
+
+:::
+
+## Overview
+
+In the process to gather metrics on our system's ability to handle increased loads and users we have performed tests to evaluate the maximum number of concurrent users a Rasa chatbot can handle with certain machine configurations.
+In each test case we spawned the following number of concurrent users at peak concurrency using a spawn rate of 1000 users per second. In our tests we used Rasa's [Http-Api](https://rasa.com/docs/rasa/pages/http-api),
+along with common opensource load testing tool such as [locust](https://locust.io/).
+
+|        Users             |               Configuration                        |
+|--------------------------|----------------------------------------------------|
+| Up to 50,000             |         6vCPU, 16 GB                               |
+| Up to 80,000             |         6vCPU, 16 GB but with almost 90% CPU usage |
```

Comment:
Can we actually have CPU usage as a separate column actually for both cases.

Reply:
Do we only have data for these two 'user counts/reference architectures'? Or am I missing something? Perhaps we can ask CSEs to give us the most common 'reference architectures' of our customers?

Reply:
@ancalita I can put CPU usage as a seperate column, I stuck to the format that was used [here](https://docs.gitlab.com/ee/administration/reference_architectures/1k_users.html)

Reply:
okay, gotcha! but I wonder if this creates value for customers in this form if our examples are not very close to their user numbers. let's do this now and try to make it more valuable for next iteration!

Reply:
Already on it, plus hoping to get more ideas on the friday meeting with the team.